.github/workflows/build_and_test_cvp_image.yml

```diff
@@ -0,0 +1,59 @@
+name: "Build and test CVP CI image"
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: "Build the cvp_image inside Dockerfiles"
+        run: |
+          podman --version
+          cd Dockerfiles 
+          podman build -t cvp_image:latest -f Dockerfile.cvp --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=master
+
+      - name: "Run example bundle through cvp_image -> POSITVE test"
+        run: |
+          cd Dockerfiles
+          mkdir output_logs
+          podman run -it -v ./example-bundle:/project/operator-bundle -v ./output_logs:/project/output --security-opt label=disable cvp_image:latest
+          # clean output logs for next step
+          rm -rf output_logs
+
+      - name: "Run example bundle through cvp_image --> negative test"
+        continue-on-error: true
+        run: |
+          # cd into Dockerfiles since it resets everystep
+          cd Dockerfiles
+          mkdir empty-example-bundle output_logs
+          podman run -it -v ./empty-example-bundle:/project/operator-bundle -v ./output_logs:/project/output --security-opt label=disable cvp_image:latest
+
+      - name: "Fail when previous step does not fail"
+        run: |
+          grep -inr "metadata not found in ../operator-bundle" Dockerfiles/output_logs/
+
+      - name: "Run example operator-metadata through cvp_image --> negative test"
+        continue-on-error: true
+        run: |
+          # cd into Dockerfiles since it resets everystep
+          cd Dockerfiles
+          mkdir test_operator_work_dir
+          podman run -it -v ./example-metadata-without-alm-annotations:/project/operator_dir -v ./output_logs:/project/output -v ./test_operator_work_dir:/project/test_operator_work_dir -e TEST_NAME=prepare_operator_metadata --security-opt label=disable cvp_image:latest
+
+      - name: "Fail when previous step does not fail"
+        run: |
+          grep -inr "Failed due to CSV.metadata.annotations\['alm-examples'\] is set to empty" Dockerfiles/output_logs
```

Comment:
We need to check whether all expected files were generated. It is not enough to check message in error log.

Moreover current approach for "running" tests does not allow local execution of tests. This is huge disadvantage.
Because you can run "tests" just in github actions.

Reply:
Local execution of tests can be avoided because whenever you fork this repository all the tests will be added to the fork. if you push anything to your fork you should be able to run it. 
However, We can run the tests locally by running the commands inside GitHub actions locally. 
There is no resitriction.